articles/step-by-step-on-how-to-user-nodejs-repl-tutorial/index.md

```diff
@@ -0,0 +1,203 @@
+### Introduction
+Node.js `Read -Evaluate-Print-Loop` is an easy-to-use command-line tool for processing Node.js expressions.
+This shell captures the user's JavaScript code inputs, interprets and evaluates the result of this code, displays the result to the screen, and repeats the process till the user quits the shell.  
+
+It's therefore, important to note that this tool does not require file creation to write codes. It comes with ready to use Node.js development environment.
+
+### Prerequisites
+This tutorial requires:
+1. Node.js Installed in your development local machine.
+2. Basic knowledge of JavaScript.
+
+### Getting started with REPL
+Enter the following in the terminal:
+
+```bash
+$ node
+
+```
+Output:
+
+```bash
+> 
+```
+Entering the command `node` in the terminal starts the REPL command-line tool indicated by the `>` symbol.  
+This symbol acts as an indicator that JavaScript is ready to read and evaluate your code.  
+
+We can test the functionality by printing 'hello world' using REPL as shown below:
+
+```bash
+$ node
+> let hello = 'hello world';
+undefined
+> hello
+'hello world'  // output
+> 
+
+```
+In the script, we declare a variable `hello` and assign the text 'hello world'.  
+Next, we call this variable and press enter, to print the 'hello world' string on the screen. That's it.  
+
+Now how do we exit REPL?
+To exit this tool, press `ctrl + Z on your keyboard.  
+
+Output:
+```bash
+> let hello = 'hello world';
+undefined
+> hello
+'hello world'
+> 
+(To exit, press Ctrl+C again or Ctrl+D or type .exit)
+> 
+
+```
+As shown in the output, you can as well use `ctrl+D` or type `.exit`.  
+
+### Executing code using REPL.
+
+As discussed previously, this tool simplifies work by providing a quick way to test Node.js code without creating files.  
+> It's also important to note that any valid JavaScript code can be executed using REPL.
+
+ 1. Performing Arithmetical operations in REPL
```

Comment:
Use the markdown header `####`.

Something like this `#### Performing arithmetical operations in REPL`